src/instanceMethods/keydown-handler.js

```diff
@@ -42,18 +42,12 @@ export const setFocus = (innerParams, index, increment) => {
 }
 
 const arrowKeysNextButton = [
-  'ArrowRight', 'ArrowDown',
-  'Right', 'Down' // IE11
-]
-
-const arrowKeysPreviousButton = [
```

Comment:
You've mistakenly removed this variable instead of `escKeys` :)

Please revert this one and remove `escKeys`.